diff --git a/src/config.ts b/src/config.ts
index da52ccf..3eb7007 100644
--- a/src/config.ts
+++ b/src/config.ts
@@ -26,7 +26,7 @@ export const config = {
     secret: process.env.POSTGRES_SECRET || "secret",
   },
   server: {
-    serverPort: "3000",
+    serverPort: process.env.SERVER_PORT || "3000",
     initialUsername: process.env.INITIAL_USERNAME || "admin",
     initialPassword: process.env.INITIAL_PASSWORD || "admin",
   },
