diff --git a/src/config.ts b/src/config.ts
index 8ff71d4..c15d460 100644
--- a/src/config.ts
+++ b/src/config.ts
@@ -60,9 +60,9 @@ export const config = {
       process.env.LDAP_GROUP_SEARCHBASE || "ou=groups,dc=solawi,dc=biz",
     groupMemberAttribute: process.env.LDAP_GROUP_MEMBER_ATTRIBUTE || "member",
     groupRoleMapping: {
-      [UserRole.ADMIN]: "plantage_admin",
+      [UserRole.ADMIN]: "wfd_plantage_admin",
       [UserRole.EMPLOYEE]: "plantage_employee",
-      [UserRole.USER]: "plantage_user",
+      [UserRole.USER]: "wfd_plantage24",
     },
   },
   testing: {
